chapters/annotations.tex

```diff
@@ -584,7 +611,7 @@ \subsection{Connection Restrictions}\label{connection-restrictions}
 
 A connector component declaration may have the following annotation:
 \begin{lstlisting}[language=modelica]
-annotation(mayOnlyConnectOnce = "message");
+/*literal*/ constant String mayOnlyConnectOnce;
 \end{lstlisting}%
 \annotationindex{mayOnlyConnectOnce}
 
```

Comment:
On the line below (GitHub won't let me attach suggestion there), the test should start with:
```
Only has effect when \lstinline!true!, making it an error…
```